deploy/manifests/support-bundle-manager.yaml

```diff
@@ -4,7 +4,7 @@ kind: Deployment
 metadata:
   labels:
     app: support-bundle-manager
-    harvesterhci.io/supportbundle: sample
+    rancher/supportbundle: sample
```

Comment:
`rancher.io`

also, use `support-bundle` instead of `supportbundle`. Label names are normally separated by `-`.

Reply:
@yasker fixed